vignettes/cookbook_interpretation.Rmd

```diff
@@ -14,59 +14,164 @@ knitr::opts_chunk$set(
+    technology == "renewablescap",
+    region == "global",
+    scenario_source == "demo_2020"
+  )
+)
+```
+
+## Sectoral Decarbonization (SDA) Target
+
+[brief description of SDA metric]
+
+For detailed information about the Sectoral Decarbonization (SDA) metric and how it is calculated, see the [Metrics appendix](cookbook_metrics.html#sectoral-decarbonization-approach).
+
+### `target_sda()` function
+
+The `target_sda()` function calculates the Sectoral Decarbonization Target metric and outputs a table including the calculated targets at the portfolio level or at the company level, depending on the options selected.
+
+#### Explanation of Output
+
+The primary column/variable of interest in the output is the `emission_factor_value` variable. What the `emission_factor_value` variable represents depends on the values in all of the other columns, and in particular the `emission_factor_metric` column. If the value in the `emission_factor_metric` column on the same row is `"projected"`, then the value in the `emission_factor_value` variable will represent the physical emission intensity level of your portfolio based on the ABCD data for that sector/year/region/scenario/etc. If the value in the `emission_factor_metric` column on the same row begins with `"target_"`, e.g. `"target_cps"`, then the value in the `emission_factor_value` variable will represent the target level of physical emission intensity for that sector/year/region/scenario/etc. based on the ABCD start year production value of your portfolio modified to be relative to the scenario data for the given year. If the value in the `emission_factor_metric` column on the same row is `"corporate_economy"`, then the value in the `emission_factor_value` variable will represent [the total physical emission intensity levels of the entire sector adjusted by the start year physical emission intensity level of your portfolio, which can be used aa a benchmark to compare to your portfolio's production levels]. If the value in the `emission_factor_metric` column on the same row starts with `"adjusted_"`, then the value in the `emission_factor_value` variable will represent [?].
+
+If you use the `by_company = TRUE` option of the `target_sda()` function, most of what's said above is true except relative to the company listed in that row in the `name_abcd` column.
+
+For detailed information about the table output of the `target_sda()` function and its columns, see the [data dictionary](data_outputs.html#target_sda-output).
+
+#### Interpretation & Intuition
+
+[2 or 3 sentences about interpretation]
+
+[2 or 3 sentences about intuition]
+
+### Emission Intensity Plot
+
+The emission intensity metric measures the average CO^2^ intensity of the portfolio in the steel and cement sector. This emission intensity is given as CO^2^/economic unit of output (for example, CO^2^/per ton of steel produced). This is then compared to an emission intensity reference point set by a climate scenario.
```

Comment:
Hard coding "steel and cement" is likely to fall out of sync. 
Can you encode this programmatically using the sectors found in the `co2_intensity_scenario_demo`? 

It's also not a perfect solution, but we are more likely to remember updating that dataset if we add a new sector, then we are to remember updating this.